README.md

```diff
@@ -51,6 +51,14 @@ In the future, there may be a couple APIs that are dependent on new OS features
 however we will do our best to ensure
 this is the exception and not the norm, and provide reasonable fallbacks when possible.
 
+## Support
+
+### How to file issues and get help

```

Comment:
Please wrap markdown at 100 characters.  Also add blank lines between the heading and the content.  Consider using VSCode and the Prettier extension, which knows how to do this automatically.